amc2moodle/amc2moodle/test/QCM_wo-tikz.xml

```diff
@@ -181,5 +181,11 @@ A complicated chemical equation \(\ce{Hg^2+ -&gt;[I-] HgI2 -&gt;[I-] [Hg^{II}I4]
 
 
 
-]]></text></questiontext><answer format="html" fraction="100.0"><text><![CDATA[a simpler one \(\ce{CO2 + C -&gt; 2 CO}\).]]></text><fraction>100.0</fraction></answer><answer format="html" fraction="-50.0"><text><![CDATA[Wrong Choice! ]]></text><fraction>-50.0</fraction></answer><shuffleanswers>true</shuffleanswers><answernumbering>none</answernumbering><defaultgrade>1.0</defaultgrade></question>
+]]></text></questiontext><answer format="html" fraction="100.0"><text><![CDATA[a simpler one \(\ce{CO2 + C -&gt; 2 CO}\).]]></text><fraction>100.0</fraction></answer><answer format="html" fraction="-50.0"><text><![CDATA[Wrong Choice! ]]></text><fraction>-50.0</fraction></answer><defaultgrade>1.0</defaultgrade><shuffleanswers>true</shuffleanswers><answernumbering>none</answernumbering></question>
+  <question type="category"><category><text>$course$/test_notikz/bcl1</text></category></question>
+  <question type="numerical"><answer fraction="100"><text>290</text></answer><name><text>bcl1a</text></name><questiontext format="html"><text><![CDATA[
+Combien de fois le programme suivant affiche-t-il <tt>"x"</tt> ?
```

Comment:
I would really like to have a paragraph break here, like in the original LaTeX file.

Reply:
This is fixed by b2010b0.